### ISSUE TITLE
Replace morphology tags in editor with new from August 2018 meeting

### ISSUE THREAD

**Issue by cajosc** [spraakbanken/swell-editor#99]

**Comment by arildm** [comment]
From the [meeting minutes](https://spraakbanken.gu.se/sites/spraakbanken.gu.se/files/Minnesanteckningar_SweLL_Projektmote_20180830-31.pdf):

```
Vi ersätter existerande kategorier med följande både i guidelines och i
annoteringsverktyget
o Function: (eleven behärskar inte)
§ M-CASE (kasus, genitiv)
§ M-DEF (bestämdhet)
§ M-GEND (genus)
§ M-KOMP (komparativ)
§ M-NUM (numerus)
§ M-VERB (tempus, modus och aspekt)
---------
o Form: (fel ändelse men rätt) böjningskategori)
§ M-F (incl adverb)
---------
o M-Other
```

**Comment by arildm** [comment]
Fixed in aaac3e2
